src/game.js

```diff
@@ -20,19 +21,28 @@
 
   Game.prototype.run = function () {
     var self = this;
-    // this.update(); // change the game's state - ball moving, score, paddles moving
+    this.update(); // change the game's state - ball moving, score, paddles moving
     this.draw(); // render the updated changes
     requestAnimationFrame(function () {
       self.run()
     });
   };
+  
+  Game.prototype.update = function () {
+    this.clearCanvas();
+    this.ballXLocation += this.ballDX
```

Comment:
please move this.ballXLocation and ballYLocation to a separate function we can call here. We'll then add a similar one for paddle.